recipe/meta.yaml

```diff
@@ -7,18 +7,10 @@ package:
 source:
   fn: iris-v{{ version }}.tar.gz
   url: https://github.com/SciTools/iris/archive/v{{ version }}.tar.gz
-  sha256: 62e4cb152366e21aa845f1fbf0520abda39ccc7c5952e6a434eed5cd9f2cc140
-  patches:
-    # Issue reported in https://github.com/SciTools/iris/issues/2004.
-    # We work around this in build.sh.
-    - no_compile_setup.patch  # [osx]
```

Comment:
Thanks @ocefpaf 
I don't have access to a mac so I couldn't test it locally. So I was just going to let travis run and see what happens.

I do like the idea of a no_compile as a setup.py option so perhaps we can implement that for the next release of Iris.

I will push up the change reinstating this patch now.